ci/taos/plugins-base/pr-audit-build-android.sh

```diff
@@ -168,6 +187,42 @@ function pr-audit-build-android-run-queue(){
         popd
         echo -e "[DEBUG] The current directory: $(pwd)."
 
+        # Start to build NNStreamer API for Android.
```

Comment:
@jaeyun-jung 
When you think the PR is a final version, Please append the execution result of the below statement in PR body. 
```bash
u1604$ sudo apt install shellcheck
u1604$ shellcheck  ./pr-audit-build-android.sh
```

Reply:
added as a comment